src/components/ui/formik/validationSchemas.ts

```diff
@@ -37,6 +37,13 @@ export const validJSONSchema = Yup.string().test(
   },
 );
 
+export const pullSecretKnownOrRequired = (values: ClusterConfigurationValues) =>
+  Yup.string().test(
+    'pull-secret-known-or-required',
+    'Pull secret ust be provided.',
```

Comment:
Typo
```suggestion
    'Pull secret must be provided.',
```
Or to match the other fields message: 'Required to install the cluster.'